manifests/server.pp

```diff
@@ -546,6 +629,70 @@
     }
   }
 
+  # Template uses:
```

Comment:
I made the experience that comments like this outdate really fast and you cannot trust them... Could you remove this?